src/main/app/claims/claimStoreClient.ts

```diff
@@ -195,4 +195,19 @@ export class ClaimStoreClient {
         }
       })
   }
+
+  // This is a tactical solution until SIDAM is able to add roles to user ID
+  addRoleToUser (user: User, role: string): Promise<void> {
+    if (!user) {
```

Comment:
Please add another precondition check to reject empty roles.

Reply:
done